packages/components/resizable/src/use-resizable.ts

```diff
@@ -57,9 +57,10 @@ export interface ResizableItemControl extends ImperativePanelHandle {}
 
 interface ResizableContext {
   controlRef: RefObject<ResizableGroupControl>
-  direction: ResizableDirection
   isDisabled: boolean
+  orientation: ResizableDirection
```

Comment:
@odendayoko @uraaaa24 

Do not include documentation fixes and package fixes in the same PR.

It will cause the documentation to be updated even though the package has not yet been released.